Sign transaction refactor (#183)
onflow/flow-cli: docs/build-transactions.md

---
title: Build a Transaction with the Flow CLI
sidebar_title: Build a Transaction
description: How to build a Flow transaction from the command line
---

The Flow CLI provides a command to build a transactions with options to specify
authorizer accounts, payer account and proposer account. 

> Build command don't produce any signatures and 
> should be used with the `sign` and `send-signed` commands. 

Reviewer: ```suggestion
The `build` command doesn't produce any signatures and instead
is designed to be used with the `sign` and `send-signed` commands. 
```

Use this functionality in the following order:

> 1. Use this command to build the transaction
> 2. Use the sign command to sign with all accounts specified in the build process
> 3. Use send signed command to submit the signed transaction to the network.

Reviewer: ```suggestion
1. Use this command (`build`) to build the transaction.
2. Use the `sign` command to sign with all accounts specified in the build process.
3. Use `send-signed` command to submit the signed transaction to the Flow network.
```
